Formula/justprep.rb

```diff
@@ -0,0 +1,36 @@
+class Justprep < Formula
+  desc "Pre-processor to the 'just' command-line utility"
+  homepage "https://github.com/MadBomber/justprep"
+  url "https://github.com/MadBomber/justprep/archive/refs/tags/v1.2.3.tar.gz"
+  sha256 "0277839a9e7e3b821b2cf72efcb364a839a59dbbbaab0b1baa02594a4994f70c"
+  license "MIT"
+
+  depends_on "crystal" => :build
+  depends_on "just" => :build
+
+  depends_on "bdw-gc"
```

Comment:
```suggestion
  depends_on "just" => [:build, :test]
  depends_on "crystal" => :build
  depends_on "bdw-gc"
```
Since you need `just` in the test.

Reply:
wilco